di-ipv-orchestrator-stub/src/main/resources/data/inheritedJWTCredentialSubjects.json

```diff
@@ -0,0 +1,37 @@
+{
+  "data": [
+    {
+      "label": "Alice Jane Dow",
```

Comment:
Might be worth adding Kenneth too.

I don't think it should actually matter - we won't be correlating the names, but it might prove useful and less confusing for testing purposes if a journey can be run with the same user.